modules/common-workflows/pages/workflow-clients-update-rec-actions.adoc

```diff
@@ -0,0 +1,101 @@
+- a dedicated maintenance window exists
+
+
+== Outcome / Resolution
+
+Successful completion of this workflow results in consistent and supportable state.
+
+
+== Preparation
+
+// FIXME 4.3 backport wersion
+// Before you start, you should have a number of {salt} clients onboarded.
+Before you start, you should have a number of clients onboarded.
+It may make sense to have them sorted into groups you want to update together.
+In this workflow we use a system group named [literal]``infra-services``.
+
+
+== Step-by-Step Workflow Instructions
+
+To update a client two steps are required.
+A third step is optional but it is highly recommended to finalize the update process.
+
+
+.Procedure 1: Creating a recurring action to update {salt} itself
+[role=procedure]
+. As an example we create the action to update {salt} itself as a recurring action for all systems in the organization.
+  In the {productname} {webui} go to menu:Home[My Organization > Recurring Actions] and click btn:[Create].
+. Select [literal]``Action Type`` **Custom State** and enter a [literal]``Schedule Name`` like [literal]``update-salt``.
+. Select a schedule.
+  For example, ** Weekly: Wednesday, 9:00 am **.
+. Assign the [literal]``update-salt`` state by selecting the checkbox.
+. clicking btn:[Save Changes] to save the action.
+. You can edit the execution order of the states if needed.
+  Click btn:[Confirm] to confirm the order.
+. Click btn:[Create Schedule] to save the action.
+
+
+Procedure 2: Create a Recurring Action to Apply All Available Updates to the Systems
+[role=procedure]
+. As an example we create the action to apply all updates as a recurring action for a system group called "infra-services".
+In the {productname} {webui} go to menu:Systems[System Groups] and click on [literal]``infra-services``.
+. Now go to [literal]``Recurring Actions`` and click btn:[Create].
+. Select [literal]``Action Type`` **Custom State** and enter a [literal]``Schedule Name`` like [literal]``full-system-update``.
+. Select a Schedule. 
+  For example, ** Weekly: Wednesday, 9:30 am **.
+Keep enough time between this action and the [literal]``update-salt`` action.
+The [literal]``update-salt`` actions must be finished on all systems before this action should be executed.
+    . Assign the states [literal]``util.syncall``, [literal]``certs``, [literal]``channels`` and [literal]``uptodate` by selecting the checkboxes.
+. Save the action by clicking btn:[Save Changes].
+. You can edit the execution order of the states. 
+  The order should be [literal]``util.syncall``, [literal]``certs``, [literal]``channels``, and finally [literal]``uptodate``.
+Click btn:[Confirm] to store the order.
+. Click btn:[Create Schedule] to save the action.
+
+    The [literal]``uptodate` also performs a reboot, if an update requests it. 
+    This reboot be prevented if needed. 
+    For example, the systems which use live patching need to prevent the rebooting following the update.
```

Comment:
```suggestion
The [literal]``uptodate` also performs a reboot, if an update requests it. 
This reboot be prevented if needed. 
For example, the systems which use live patching need to prevent the rebooting following the update.
```